packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -497,6 +497,13 @@
         <protected>false</protected>
         <shortDescription>Apply</shortDescription>
     </labels>
+    <labels>
+        <fullName>quantic_TimeframeInputApply</fullName>
+        <value>Apply custom timeframe values for the {{0}} facet</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Apply custom timeframe values for the [component label] facet</shortDescription>
```

Comment:
Be careful with the length of the `shortDescription` value. It is limited to something like 80 characters. You won't be able to deploy the label if the `shortDescription` is too long.

Reply:
Thanks for info!!!